schema_poc/test_producer.py

```diff
@@ -1,40 +1,76 @@
 from datetime import datetime
-from schema_registry import SchemaRegistry
+from os import getenv
+
 from producer import Producer
+from schema_registry import SchemaRegistry
 
 # Before running this test, the schema is going to need to be loaded into RedPanda schema registry.
 # The test here assume RedPanda is running on local host port 8081, which it will be if you used the dependencies
 # Docker Compose file.  Then you need to have inserted the schema for this message type.  This can be done with PostMan
-# by creating a POST request to http://localhost:8081/subjects/plate-map-sample/versions with the JSON body shown.
-# Replace the schema definition with a slash escaped string based on the .avsc file in this directory.
+# by creating a POST request to http://localhost:8081/subjects/{subject_name}/versions with the JSON body shown.
+# Replace {subject_name} in the URL with the intended subject name.  See the keys on MESSAGES below.
+# Replace the schema definition with a slash escaped string based on the relevant .avsc file in this directory.
 # {
 #     "schema": "slash_escaped_string_of_schema_json"
 # }
 
-sample = {
-    "labId": "CPTD",
-    "sampleUuid": "UUID-123456-01",
-    "plateBarcode": "BARCODE001",
-    "rootSampleId": "R00T-S4MPL3-1D",
-    "plateCoordinate": "A6",
-    "result": "positive",
-    "preferentiallySequence": True,
-    "mustSequence": True,
-    "fitToPick": True,
-    "testedDateUtc": datetime(2022, 2, 1, 13, 45, 8),
+create_message = {
+    "messageUuid": "UUID-789012-23",
+    "messageCreateDateUtc": datetime.utcnow(),
+    "plate": {
+        "labId": "CPTD",
+        "plateBarcode": "BARCODE001",
+        "samples": [
+            {
+                "sampleUuid": "UUID-123456-01",
+                "rootSampleId": "R00T-S4MPL3-01",
+                "plateCoordinate": "A6",
```

Comment:
@emrojo Do we want A06 or A6 for coordinates? Best to match in the example test.